optaweb-tsp-ui/pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.optaweb.tsp</groupId>
+	<artifactId>optaweb-tsp-ui</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>optaweb-tsp-ui</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
This pom should extend the root pom. Similar to how optaweb-employee-rostering-shared has a parent optaweb-employee-rostering. When that parent is there, all these properties should be moved there, see the root pom of optaweb-employee-rostering